app/components/user-dropdown.tsx

```diff
@@ -25,10 +26,12 @@ export function UserDropdown() {
 						onClick={(e) => e.preventDefault()}
 						className="flex items-center gap-2"
 					>
-						<img
+						<Img
 							className="h-8 w-8 rounded-full object-cover"
 							alt={user.name ?? user.username}
 							src={getUserImgSrc(user.image?.id)}
+							width={416}
+							height={416}
```

Comment:
I was pretty conservative with these max-width values to have all images look and feel high quality and not pixelated. Note that the CSS width and height will override the HTML attributes, so there are no visual changes. The width & height attributes are still important for the browser to compute the aspect ratio and for openimg to request a good-enough resized image.

Reply:
Honestly, that's probably what I would have done too. 👍

Reply:
Actually, I think that this is going to be 256x256, and I think that we will get plenty of high-quality images by setting it to `256`.